public/templates/templates.go

```diff
@@ -99,9 +116,173 @@ func (t *Templater) RenderTemplate(w io.Writer, data interface{}, mainTmpl Templ
 		"len": func(a string) int {
 			return len(a)
 		},
+		"lenA": func(a []string) int {
+			return len(a)
+		},
+		"memberAddPermission": func(id int) bool {
+			u, err := t.User.GetUser(context.Background(), user.User{UserID: id})
+			if err != nil {
+				fmt.Println(err)
+				return false
+			}
+
+			p, err := t.User.GetPermissionsForUser(context.Background(), u)
+			if err != nil {
+				fmt.Println(err)
+				return false
+			}
+
+			for _, perm := range p {
+				if perm == permissions.ManageMembersMembersAdd.GetString() {
+					return true
+				}
+			}
+			return false
+		},
```

Comment:
This seems like it does a lot of the same as `checkPermission` unless I'm missing something?

Reply:
This is removed now